CHANGELOG.md

```diff
@@ -12,7 +12,11 @@ All notable changes to this project will be documented in this file.
 
 ## [3.5.3] - 2023-02-18
 ### Changed
-- Added `undo` subcommand, introducing Undo scripts
+- Added `undo` subcommand, introducing Undo script
+
+## [3.5.4] - 2024-02-05
```

Comment:
[This commit](https://github.com/salemove/schemachange/pull/4/commits/2a4b6211c5f771bdf150e41950cb16ab095e9818) already has 3.5.4 version, please bump it because otherwise docker will continue to use cached 3.5.4 without your changes.